pkg/api/pod/util.go

```diff
@@ -331,6 +331,28 @@ func usesHugePagesInProjectedEnv(item api.Container) bool {
 	return false
 }
 
+// useSysctlRegexContainSlashValidation return value is determined by the following:
+// 1.use strict validation for Create requests, and for Update requests where the existing object's sysctl does not contain a forward slash, return false
+// 2.use relaxed validation for Update requests where the existing object's sysctl contains a forward slash, return true
+func useSysctlRegexContainSlashValidation(podSpec *api.PodSpec) bool {
+	if podSpec == nil {
+		return false
+	}
+	securityContext := podSpec.SecurityContext
+	if securityContext != nil {
+		if len(securityContext.Sysctls) != 0 {
+			for _, s := range securityContext.Sysctls {
+				if len(s.Name) == 0 {
+					return false
+				} else if strings.Contains(s.Name, "/") {
+					return true
+				}
+			}
+		}
```

Comment:
no need to check length of Sysctls or Name, simplify to
```go
for _, s := range securityContext.Sysctls {
  if strings.Contains(s.Name, "/") {
    return true
  }
}
return false
```

Reply:
Updated.